js/coinex.js

```diff
@@ -614,6 +615,81 @@ module.exports = class coinex extends Exchange {
         return this.parseOrder (data, market);
     }
 
+    async cancelAllOrders (symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancellAllOrders() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const marketId = market['id'];
+        let accountId = this.safeString2 (params, 'id', 'account_id');
+        if (accountId === undefined) {
+            accountId = '0'; // default to main account, note: account id is a required parameter, margin account ID's can be found by calling the getMarginAccount method
+        }
+        const request = {
+            'account_id': accountId, // main account ID: 0, margin account ID: See < Inquire Margin Account Market Info >, future account ID: See < Inquire Future Account Market Info >
+            'market': marketId,
+        };
+        const response = await this.privateDeleteOrderPending (this.extend (request, params));
+        //
+        // {"code": 0, "data": null, "message": "Success"}
+        //
+        return response;
+    }
+
+    async getMarginAccount (params = {}) {
+        const symbol = this.safeString (params, 'market');
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' getMarginAccountId() requires a market argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const marketId = market['id'];
+        const request = {
+            'market': marketId,
+        };
+        params = this.omit (params, 'market');
+        const response = await this.privateGetMarginAccount (this.extend (request, params));
```

Comment:
Looking at the response, I would suggest removing this method and maybe moving it to `fetchBalance()` in another PR.